ACE/ace/config-win32-common.h

```diff
@@ -55,8 +55,8 @@
 #  endif  /* !_FILE_OFFSET_BITS */
 #endif /* _WIN64 || WIN64 */
 
-#if !defined (_WIN32_WINNT)
-# define _WIN32_WINNT 0x0501 // pretend it's at least Windows XP or Win2003
+#if !defined (_WIN32_WINNT) || _WIN32_WINNT < 0x0600
+#  error _WIN32_WINNT must be at least 0x0600
```

Comment:
According to @jwillemsen's comment should this line be changed to
`# define ACE_LACKS_IF_NAME_INDEX`
If were doing something like that, this block would have to move to after the `#include` of windows headers to pick up the default value of _WIN32_WINNT